plugin/src/main/java/com/denizenscript/denizen/scripts/commands/entity/FakeInternalDataCommand.java

```diff
@@ -0,0 +1,96 @@
+package com.denizenscript.denizen.scripts.commands.entity;
+
+import com.denizenscript.denizen.nms.NMSHandler;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.Utilities;
+import com.denizenscript.denizencore.DenizenCore;
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsRuntimeException;
+import com.denizenscript.denizencore.objects.core.DurationTag;
+import com.denizenscript.denizencore.objects.core.MapTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.denizencore.scripts.commands.generator.*;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Player;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FakeInternalDataCommand extends AbstractCommand {
+
+    public FakeInternalDataCommand() {
+        setName("fakeinternaldata");
+        setSyntax("fakeinternaldata [entity:<entity>] [data:<map>|...] (for:<player>|...) (speed:<duration>)");
+        setRequiredArguments(3, 4);
+        autoCompile();
+    }
+
+    // <--[command]
+    // @Name FakeInternalData
+    // @Syntax fakeinternaldata [entity:<entity>] [data:<map>|...] (for:<player>|...) (speed:<duration>)
+    // @Required 3
+    // @Maximum 4
+    // @Short Sends fake entity data updates, optionally animating them with sub-tick precision.
+    // @Group entity
+    //
+    // @Description
+    // Sends fake internal entity data updates, optionally sending multiple over time.
+    // This supports sub-tick precision, allowing smooth/high FPS animations.
+    //
+    // The input to 'data:' is a list of <@link object MapTag>s, with each map being a frame to send, with each map being formatted like <@link mechanism EntityTag.internal_data>'s input.
+    //
+    // Optionally specify a list of players to fake the data for, defaults to the linked player.
+    //
+    // 'speed:' is the amount of time between each frame getting sent, supporting sub-tick delays.
+    //
+    // @Usage
+    // Animates an item display entity's item for the linked player, and slowly scales it up.
+    // - fakeinternaldata entity:<[item_display]> data:[item=iron_ingot;scale=0.6,0.6,0.6]|[item=gold_ingot;scale=0.8,0.8,0.8]|[item=netherite_ingot;scale=1,1,1] speed:0.5s
+    //
+    // @Usage
+    // Animates a rainbow glow on a display entity for all online players.
+    // - define color <color[red]>
+    // - repeat 361 from:0 as:hue:
```

Comment:
A: hue is 0-255, not 0-360?
B: even if it wasn't a byte, the usage of `361` would still be 1 too many here (in a circle, 0 degrees and 360 degrees are the same value)

Reply:
Should be corrected now